src/pyseekdb/cli.py

```
@@ -0,0 +1,311 @@
+                            if doc and len(doc) > 50:
+                                doc = doc[:47] + "..."
+                            table.add_row(str(id_), str(doc), str(meta))
+
+                        console.print(table)
+                    else:
+                        console.print("[yellow]No records found.[/yellow]")
+                except Exception as e:
+                    console.print(f"[red]Error peeking records:[/red] {e}")
+    except SystemExit:
+        raise
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)